mrcp.go

```diff
@@ -108,8 +108,11 @@ type SynthResult struct {
 func (a *AGI) MRCPSynth(prompt string, opts string) (res *SynthResult, err error) {
 	var cause string
 	res = new(SynthResult)
-
-	ret, err := a.Exec([]string{"MRCPSynth", prompt, opts}...)
+	execOpts := []string{
+		fmt.Sprintf(`"%s"`, prompt),
+		opts,
+	}
+	ret, err := a.Exec([]string{"MRCPSynth", strings.Join(execOpts, ",")}...)
```

Comment:
We shouldn't need to declare a `[]string{}` and then expand it (`...`), should we?
Maybe just
```
ret, err := a.Exec("MRCPSynth", strings.Join(execOpts, ","))
```
?